DUI3-DX/Converters/ArcGIS/Speckle.Converters.ArcGIS3/Features/GisFeatureToHostConverter.cs

```diff
@@ -0,0 +1,48 @@
+using ArcGIS.Core.Geometry;
+using ArcGIS.Desktop.Mapping;
+using Objects.GIS;
+using Speckle.Converters.Common;
+using Speckle.Converters.Common.Objects;
+using Speckle.Core.Models;
+
+namespace Speckle.Converters.ArcGIS3.Features;
+
+public class GisFeatureToHostConverter : IRawConversion<Base, ArcGIS.Core.Geometry.Geometry>
```

Comment:
Is it worth having some shorthand for the ArcGIS.Core.Geometry.Geometry like we are in other places?

Reply:
good idea!